docs/modules/deploy/pages/deploying-on-aws.adoc

```diff
@@ -300,6 +306,9 @@ clientConfig.getNetworkConfig().getAwsConfig()
 --
 ====
 
+Hazelcast Client on ECS can discover the members running on EC2 instances. For this you need to specify the needed <<ec2_properties,EC2 related properties>> (`iam-role`, `security-group-name`) and make sure to `ec2:DescribeInstances` permission.
+If none of the ECS or EC2 related properties are specified, the AWS discovery will try to discover the ECS memebr first, and then, if no ECS members found, it will try to discover EC2 members.
```

Comment:
If none of the ECS or EC2 related properties are specified, the AWS discovery tries to discover the ECS members. If none is found, then it tries to discover EC2 members.